internal/observability/instance.go

```diff
@@ -16,7 +16,7 @@ func GetOrCreateInstanceID() string {
 
 	if _, err := os.Stat(filePath); os.IsNotExist(err) {
 		id := uuid.New().String()
-		if err := os.WriteFile(filePath, []byte(id), 0644); err != nil {
+		if err := os.WriteFile(filePath, []byte(id), 0600); err != nil {
```

Comment:
Who will be the owner of this file? Do you think we should allow the existing more relaxed permissions here so that it's easier to read it?

Reply:
Ideally the only the process which creates should be able to read the file i.e DiceDB proc in this case. Also not sure if `UUID` stored in this file make sense for any other proc or user to be allowed access to.